controllers/childGuidance.js

```diff
@@ -0,0 +1,154 @@
+import childModel from '../models/childModel.js';
+
+const createGuidance = async (req, res) => {
+  try {
+    const data = req.body;
+
+    const guideInstance = new childModel({
+      name: data.name,
```

Comment:
@robsdagreat  I keep asking you, what is the usage of name here? please fix this, last feedback about this comment!

Reply:
Actually I changed it last time but maybe I forgot to save. I'll make sure to check it was fixed this time